src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -643,11 +635,16 @@ void FancyZones::ToggleEditor() noexcept
     params += std::to_wstring(spanZonesAcrossMonitors) + divider; /* Span zones */
         
     std::vector<std::pair<HMONITOR, MONITORINFOEX>> allMonitors;
-    allMonitors = FancyZonesUtils::GetAllMonitorInfo<&MONITORINFOEX::rcWork>();
+
+    OnThreadExecutor dpiSystemThread;
+    dpiSystemThread.submit(OnThreadExecutor::task_t{ [&] {
+                        SetThreadDpiAwarenessContext(DPI_AWARENESS_CONTEXT_SYSTEM_AWARE);
+                        allMonitors = FancyZonesUtils::GetAllMonitorInfo<&MONITORINFOEX::rcWork>();
+                     } })
+        .wait();

```

Comment:
```suggestion
   allMonitors = FancyZonesUtils::GetAllMonitorInfo<&MONITORINFOEX::rcWork>();
```
This thread is already DPI aware, no need to use `OnThreadExecutor`, just call `GetAllMonitorInfo` directly